pkg/machinery/config/encoder/documentation.go

```diff
@@ -74,8 +75,28 @@ func (d *Doc) Describe(field string, short bool) string {
 
 // Example represents one example snippet for a type.
 type Example struct {
-	Name  string
-	Value interface{}
+	sync.Once
```

Comment:
The same problem with embedding – one can call `e.Do(…)`. That's an implementation detail that should not be exposed; that should be a regular field.